lib-cloudsdk/src/main/java/com/facebook/openwifi/cloudsdk/StateInfo.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.cloudsdk;
+
+import com.facebook.openwifi.cloudsdk.models.ap.State;
+
+public class StateInfo extends State {
+	/**
+	 * Unix time in milliseconds (ms). This is added it because {@link State} is an unknown
+	 * time reference.
```

Comment:
nit: be more specific and write "State.unit.localtime" (without the `@link` annotation is fine)